README.md

```
@@ -36,6 +36,46 @@ cd vibe && cp .env.example .env
+## Features
+
+Vibe Browser includes intelligent AI-powered features:
+
+- **Memory Awareness**: Intelligent context and memory of all websites you visit
+- **Gmail Integration**: AI-powered email management and automation
+
+<details>
+<summary><strong>Gmail Setup</strong></summary>
+
+
+#### Gmail Setup
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)